spec/lib/zendesk_apps_tools/translate_spec.rb

```diff
@@ -101,6 +101,39 @@
     end
   end
 
+  describe '#write_json' do
+    let(:translate) { ZendeskAppsTools::Translate.new }
+    let(:test_file_path) { '/tmp/test.json' }
+
+    before do
+      allow(translate).to receive(:options) { { unattended: true } }
+    end
+
+    after do
```

Comment:
that way, if for some reason the test fails to create the file we don't also get errors in the after block, and we don't have to write an existence check.